charts/super-agent/README.md

```diff
@@ -2,7 +2,7 @@
 
 # super-agent
 
-![Version: 0.0.6-beta](https://img.shields.io/badge/Version-0.0.6--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 0.0.8-beta](https://img.shields.io/badge/Version-0.0.8--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
```

Comment:
I would remove This lines o we do not have to change the version each release: https://github.com/newrelic/helm-charts/blob/c48438effaf48fbc590c7acbc3c4097f4931e770/charts/super-agent/README.md.gotmpl#L6

Reply:
sure! it makes sense!